### ISSUE TITLE
DataParalle

### ISSUE THREAD

**Issue by xiaolinpan** [rusty1s/pytorch_geometric#1388]
## ❓ Questions & Help

Hi, if i want to use `DataParalle`, should i use `Global BatchNormalization` or just use `BatchNormalization`?
